apps/admin/frontend/src/app.test.tsx

```diff
@@ -204,20 +182,18 @@ test('configuring with a demo election definition', async () => {
   // You can not save as CDF when there is no election.
   expect(screen.queryAllByText('No Log File Present')).toHaveLength(0);
 
-  fireEvent.click(screen.getByText('Definition'));
+  userEvent.click(screen.getByText('Definition'));
   await screen.findByText('Load Demo Election Definition');
 });
 
 test('configuring with a VXF election definition from a file', async () => {
```

Comment:
The tests you recently added @jonahkagan get moved to the backend in a later commit, since the backend is now where parsing takes place.